resource_templates/iam_policy.tf

```diff
@@ -0,0 +1,29 @@
+// Update Tanzu Mission Control iam policy entry
+resource "tanzu-mission-control_iam" "update_iam" {
```

Comment:
please add this file when the resource is implemented and tested. Then it will reflect the latest and working example

Reply:
Sure, removed it from the PR